### PR TITLE
Added structured metrics metamodel

### PR DESCRIPTION
**Pull request by tbezierslafosse**
- Followed 1.2 beta cmof specification available [https://www.omg.org/spec/SMM/20171101/SMM.cmof](here)
- Operations have not been implemented yet

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -80,6 +80,7 @@
         <module>emf-trace</module>
         <module>emf-types</module>
         <module>emf-widespread</module>
+        <module>smm</module>
```
Comment (sunye) [inline pom.xml:83]: Rule (that should be written someday): use "emf" as a prefix for models that use the standard EMF code generation

File: smm/README.md [hunk 2]
```diff
@@ -55,8 +55,18 @@ Represent the relationship between Measure categories, and other Measure element
 **TimeStamp Primitive Type**
 This primitive represents a point in time, such as the combination of a date and a time within a day.
 ## Extensions
+SMM provides a simple extension mechanisms that provide a uniform meta-model pattern for extending SMM. It allows ad-hoc users attributes and annotations definitions to instances of SMM Elements.
```
Comment (sunye) [inline smm/README.md]: +SMM provides a simple extension mechanism that provide a uniform meta-model pattern for extending SMM. 
_It allows ad-hoc user attributes and annotations definitions to instances of SMM Elements._ (missing verb in the sentence: allows users to do what?)

**Review by sunye — APPROVED** [review]
No comments.

File: pom.xml (hunk 1 above)
Comment (tbezierslafosse) [reply pom.xml:83]: Good point, I'll change that.

**Pull request merged**